packages/components/src/styles/hidden-links/hidden-links.stories.ts

```diff
@@ -0,0 +1,192 @@
+import '../../solid-components';
+
+import { storybookDefaults, storybookHelpers, storybookTemplate } from '../../../scripts/storybook/helper';
+
+const { argTypes, parameters } = storybookDefaults('sd-hidden-links');
+const { overrideArgs } = storybookHelpers('sd-hidden-links');
+const { generateTemplate } = storybookTemplate('sd-hidden-links');
+
+// HTML helper to get syntax highlighting and formatting in the template string
+const html = String.raw;
+
+/**
+ * Hidden links can be used to show links only for keyboard users.
+ */
+
+export default {
+  title: 'Styles/sd-hidden-links',
+  component: 'sd-hidden-links',
+  parameters: {
+    ...parameters,
+    chromatic: {
+      disableSnapshot: true
+    }
+  },
+  args: overrideArgs({ type: 'slot', name: 'default', value: 'Lorem Ipsum' }),
+  argTypes
+};
+
+/**
+ * Per default the hidden links are optimized for a single `<sd-button>`.
```

Comment:
Delete default section ... that’s already shown at the top of the page.

Reply:
Will be hidden, when the docs PR is merged.